climada/engine/unsequa/calc_base.py

```diff
@@ -515,6 +585,9 @@ def _calc_sens_df(method, problem_sa, sensitivity_kwargs, param_labels, X, unc_d
         sens_second_order_dict[submetric_name] = sens_second_order
 
     sens_first_order_df = pd.DataFrame(sens_first_order_dict, dtype=np.number)
+    # Assume sens_first_order_dict is a dictionary where values are lists/arrays of varying lengths
+    #sens_first_order_df = pd.DataFrame({k: pd.Series(v, dtype=object) for k, v in sens_first_order_dict.items()}) #!for some reason this make the plotting methods fail
```

Comment:
Too long lines. Please break.
```suggestion
    #Assume sens_first_order_dict is a dictionary where values are lists/arrays of varying lengths
    #sens_first_order_df = pd.DataFrame({k: pd.Series(v, dtype=object) for k, v in sens_first_order_dict.items()}) #!for some reason this make the plotting methods fail
```

Reply:
Why is this line commented?

Reply:
I think sens_first_order_df = pd.DataFrame({k: pd.Series(v, dtype=object) for k, v in sens_first_order_dict.items()}) was added in a previous commit by @simonameiler. I commented it back because it seemed to create issues with the plot_sensitivity method (empty Axes not displaying anything). I think this commented line can be removed. @simonameiler what do you think?